app.rb

```diff
@@ -0,0 +1,179 @@
+require_relative './book'
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+    @num = 0
+  end
+
+  def start
+    puts 'Welcome to School Library!'
+    menu while @num != 7
+  end
+
+  private
+
+  def menu
+    until @num < 7 && @num.positive?
+      puts "Please choose an option by entering a number\n1- List all books\n2- List all people\n3- Create a person
+4- Create a book\n5- Create a rental\n6- List all rentals\n7- exit"
+      @num = gets.chomp.to_i
+    end
+    case @num.to_i
+    when 1 then list_books_flow
+    when 2 then list_people_flow
+    when 3 then add_person
+    when 4 then create_book
+    when 5 then create_rental
+    when 6 then list_rentals
+    when 7 then puts 'exiting'
+    end
+  end
+
+  def add_person
+    input = ''
+    until input.to_i.positive? && input.to_i < 3
+      puts 'Do you want to create a student(1) or a teacher(2)?[input the number]'
+      input = gets.chomp
+      input == '1' ? create_student : create_teacher
+    end
+    @num = 0
+  end
+
+  def create_student
+    age = 0
+    permision = true
+    input = ''
+    until age.positive?
+      puts 'age:'
+      age = gets.chomp.to_i
+    end
+    puts 'name:'
+    name = gets.chomp
+    if age < 18
+      until %w[Y N].include?(input)
+        puts 'has parents permission?[Y,N]'
+        input = gets.chomp.upcase
+        permision = input == 'Y'
+      end
+    end
+    student = Student.new(nil, age, name, parent_permission: permision)
+    @people.push(student)
+    puts 'Student created succesfuly!'
+    reset
+  end
```

Comment:
- [ ] Create a `student` method doesn't work as expected, kindly fix it.

![image](https://user-images.githubusercontent.com/55635977/164778546-7580ae66-9436-4aa7-8a86-91560cdcc883.png)